index.js

```diff
@@ -25,14 +25,14 @@ import remarkLintNoShellDollars from "remark-lint-no-shell-dollars";
 import remarkLintNoTableIndentation from "remark-lint-no-table-indentation";
 import remarkLintNoTabs from "remark-lint-no-tabs";
 import remarkLintNoTrailingSpaces from "remark-lint-no-trailing-spaces";
-import remarkLintNodejsLinks from "./remark-lint-nodejs-links.js";
-import remarkLintNodejsYamlComments from "./remark-lint-nodejs-yaml-comments.js";
 import remarkLintProhibitedStrings from "remark-lint-prohibited-strings";
 import remarkLintRuleStyle from "remark-lint-rule-style";
 import remarkLintStrongMarker from "remark-lint-strong-marker";
 import remarkLintTableCellPadding from "remark-lint-table-cell-padding";
 import remarkLintTablePipes from "remark-lint-table-pipes";
 import remarkLintUnorderedListMarkerStyle from "remark-lint-unordered-list-marker-style";
+import remarkLintNodejsYamlComments from "./remark-lint-nodejs-yaml-comments.js";
+import remarkLintNodejsLinks from "./remark-lint-nodejs-links.js";
```

Comment:
I understand the logic of moving these to the end, but previously the entire list had been in alphabetical order, which makes it very easy to determine if something has or has not been added with visual inspection. Now we effectively have two lists. I guess I'm -0 on this, but if we are going to do it, perhaps we can add a blank line between the lists to make it visually clear where one alphabetical list/section ends and the other one begins?

Reply:
I reverted this since it's not enforce anyway :)